src/scripts/main.js

```diff
@@ -1,3 +1,211 @@
 'use strict';
 
-// write code here
+const body = document.body;
+const tableBody = document.querySelector('tbody');
+const tableHead = document.querySelector('thead');
+
+const minNameLength = 4;
+const minPositionLength = 4;
+const minAge = 18;
+const maxAge = 90;
+
+let sortedAscendingOrder = false;
```

Comment:
булевые переменные лучше называть с приставкой 'is'
```suggestion
let isSortedAscendingOrder = false;
```